### PR TITLE
Graphical Update for V 2.0

### PR DESCRIPTION
**Pull request by int5749**
- Re-Structuring of Sitemap
- Enhancement of visibility for Items in Sitemap according to Status
- Transformation to Display Days-Hours-Minutes rather than Minutes only for Time Items

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -523,20 +587,43 @@ then
...
 Several Icons have been created in order to suit requirements for robo-mower. Please feel free to download from this repository and place/extract them into ..\conf\icons\classic
 
+#### Note: Worx Support confirmed private usage of their logo and picture is free
+
 1. [Distance](/openhab-conf/icons/distance.zip)
 2. [Zones](/openhab-conf/icons/zones.zip)
 3. [Refresh](/openhab-conf/icons/refresh.png)
-4. [Landroid](/openhab-conf/icons/landroid.png)
-
+4. [Worx-Logo](/openhab-conf/icons/worx_landroid_logo.png)
```
Comment (nibi79) [inline README.md:624]: please change icon back

File: README.md [hunk 2]
```diff
@@ -260,255 +262,317 @@ Bridge worxlandroid:worxlandroidBridge:MyWorxBridge "MyWorx Bridge" [ webapiUser
...
-            Text item=LandroidYaw
         }
-   }
+    }
 }
 ```
 
+### Preview of above Sitemap
+
+<img src="images/SC_BasicUI_Main_Status.png" width="50%">
```
Comment (nibi79) [inline README.md:554]: please change icon

File: README.md [hunk 3]
```diff
@@ -260,255 +262,317 @@ Bridge worxlandroid:worxlandroidBridge:MyWorxBridge "MyWorx Bridge" [ webapiUser
...
+Switch LandroidLock               "Lock"                  <mlock>           {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:common#lock"}
+
+String LandroidMacAdress          "MAC [%s]"              <text>            {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:datCommon#macAdress"}
+String LandroidSerialNumber       "Serial Number [%s]"    <text>            {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgCommon#serialNumber"}
+Number LandroidFirmware           "Firmware [v%s]"        <text>            {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:datCommon#firmware"}
+Switch LandroidOnline             "Onlinestatus [%s]"     <network>         {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:common#online"}
+Number LandroidId                 "Id []"                                   {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgCommon#id"}
+Number LandroidProductId          "Id []"                                   {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgCommon#id"}
```
[the remark this answers is not in the document]
Comment (int5749) [reply README.md:292]: 1x removed

File: README.md [hunk 4]
```diff
@@ -260,255 +262,317 @@ Bridge worxlandroid:worxlandroidBridge:MyWorxBridge "MyWorx Bridge" [ webapiUser
...
-Number          LandroidScheduleMondayStartHour         "Start Hour [%d]"                   <time>                  {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScMonday#scheduleStartHour"}
-Number          LandroidScheduleMondayStartMinutes      "Start Minutes [%d]"                <time>                  {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScMonday#scheduleStartMinutes"}
-Number          LandroidScheduleMondayDuration          "Duration [%d]"                     <time>                  {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScMonday#scheduleDuration"}
-Switch          LandroidScheduleMondayEdgecut           "Edgecut"                           <lawnmower>             {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScMonday#scheduleEdgecut"}
+Switch LandroidScheduleMondayEnable        "Monday: [MAP(landroid_schedule_enable.map):%s]"      <time>        {channel="worxlandroid:mower:MyWorxBridge:30173502170118050389:cfgScMonday#enable",autoupdateautoupdate="false"}
```
[the remark this answers is not in the document]
Comment (int5749) [reply README.md:348]: copy & replace issue => corrected

File: README.md [hunk 5]
```diff
@@ -260,255 +262,317 @@ Bridge worxlandroid:worxlandroidBridge:MyWorxBridge "MyWorx Bridge" [ webapiUser
...
-Number          LandroidScheduleTuesdayStartHour        "Start Hour [%d]"                   <time>                  {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScTuesday#scheduleStartHour"}
-Number          LandroidScheduleTuesdayStartMinutes     "Start Minutes [%d]"                <time>                  {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScTuesday#scheduleStartMinutes"}
-Number          LandroidScheduleTuesdayDuration         "Duration [%d]"                     <time>                  {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScTuesday#scheduleDuration"}
-Switch          LandroidScheduleTuesdayEdgecut          "Edgecut"                           <lawnmower>             {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScTuesday#scheduleEdgecut"}
+Switch LandroidScheduleTuesdayEnable       "Tuesday: [MAP(landroid_schedule_enable.map):%s]"      <time>        {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScTuesday#enable",autoupdateautoupdate="false"}
```
[the remark this answers is not in the document]
Comment (int5749) [reply README.md:355]: corrected

File: README.md [hunk 6]
```diff
@@ -260,255 +262,317 @@ Bridge worxlandroid:worxlandroidBridge:MyWorxBridge "MyWorx Bridge" [ webapiUser
...
-Number          LandroidScheduleWednesdayStartHour      "Start Hour [%d]"                   <time>                  {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScWednesday#scheduleStartHour"}
-Number          LandroidScheduleWednesdayStartMinutes   "Start Minutes [%d]"                <time>                  {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScWednesday#scheduleStartMinutes"}
-Number          LandroidScheduleWednesdayDuration       "Duration [%d]"                     <time>                  {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScWednesday#scheduleDuration"}
-Switch          LandroidScheduleWednesdayEdgecut        "Edgecut"                           <lawnmower>             {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScWednesday#scheduleEdgecut"}
+Switch LandroidScheduleWednesdayEnable     "Wednesday: [MAP(landroid_schedule_enable.map):%s]"      <time>        {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScWednesday#enable",autoupdateautoupdate="false"}
```
[the remark this answers is not in the document]
Comment (int5749) [reply README.md:362]: corrected

File: README.md [hunk 7]
```diff
@@ -260,255 +262,317 @@ Bridge worxlandroid:worxlandroidBridge:MyWorxBridge "MyWorx Bridge" [ webapiUser
...
-Number          LandroidScheduleThursdayStartHour       "Start Hour [%d]"                   <time>                  {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScThursday#scheduleStartHour"}
-Number          LandroidScheduleThursdayStartMinutes    "Start Minutes [%d]"                <time>                  {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScThursday#scheduleStartMinutes"}
-Number          LandroidScheduleThursdayDuration        "Duration [%d]"                     <time>                  {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScThursday#scheduleDuration"}
-Switch          LandroidScheduleThursdayEdgecut         "Edgecut"                           <lawnmower>             {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScThursday#scheduleEdgecut"}
+Switch LandroidScheduleThursdayEnable      "Thursday: [MAP(landroid_schedule_enable.map):%s]"      <time>        {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScThursday#enable",autoupdateautoupdate="false"}
```
[the remark this answers is not in the document]
Comment (int5749) [reply README.md:369]: corrected

File: README.md [hunk 8]
```diff
@@ -260,255 +262,317 @@ Bridge worxlandroid:worxlandroidBridge:MyWorxBridge "MyWorx Bridge" [ webapiUser
...
-Number          LandroidScheduleSaturdayStartHour       "Start Hour [%d]"                   <time>                  {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScSaturday#scheduleStartHour"}
-Number          LandroidScheduleSaturdayStartMinutes    "Start Minutes [%d]"                <time>                  {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScSaturday#scheduleStartMinutes"}
-Number          LandroidScheduleSaturdayDuration        "Duration [%d]"                     <time>                  {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScSaturday#scheduleDuration"}
-Switch          LandroidScheduleSaturdayEdgecut         "Edgecut"                           <lawnmower>             {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScSaturday#scheduleEdgecut"}
+Switch LandroidScheduleSaturdayEnable      "Saturday: [MAP(landroid_schedule_enable.map):%s]"      <time>        {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScSaturday#enable",autoupdateautoupdate="false"}
```
[the remark this answers is not in the document]
Comment (int5749) [reply README.md:383]: corrected

File: README.md [hunk 9]
```diff
@@ -260,255 +262,317 @@ Bridge worxlandroid:worxlandroidBridge:MyWorxBridge "MyWorx Bridge" [ webapiUser
...
-Number          LandroidAllocation6                     "Alloction 6 []"                    <zones>                 {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgMultiZones#allocation6"}
-Number          LandroidAllocation7                     "Alloction 7 []"                    <zones>                 {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgMultiZones#allocation7"}
-Number          LandroidAllocation8                     "Alloction 8 []"                    <zones>                 {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgMultiZones#allocation8"}
-Number          LandroidAllocation9                     "Alloction 9 []"                    <zones>                 {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgMultiZones#allocation9"}
+Switch LandroidScheduleSundayEnable        "Sunday: [MAP(landroid_schedule_enable.map):%s]"      <time>        {channel="worxlandroid:mower:MyWorxBridge:MySerialNumber:cfgScSunday#enable",autoupdateautoupdate="false"}
```
[the remark this answers is not in the document]
Comment (int5749) [reply README.md:390]: last copy&replace => corrected

File: README.md [hunk 10]
```diff
@@ -260,255 +262,317 @@ Bridge worxlandroid:worxlandroidBridge:MyWorxBridge "MyWorx Bridge" [ webapiUser
...
 ```
 
 ### .sitemap
 ```
 sitemap landroid label="Landroid"
 {
-    Group item=Shaun icon="landroid" {
-        Frame {
-            Switch item=LandroidEnable label="Mowing enabled"
+    Group item=Shaun icon="worx_landroid_logo.png" {
```
[the remark this answers is not in the document]
Comment (int5749) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (int5749) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (int5749) [reply README.md:554]: new screenshot made => corrected

**Comment by int5749** [comment]
All above corrected in next Pull Request

**Pull request merged**